PEAR/Config.php

```diff
@@ -1022,15 +1022,19 @@ function _readConfigDataFrom($file)
         }
 
         $size = filesize($file);
-        $rt = get_magic_quotes_runtime();
-        set_magic_quotes_runtime(0);
+        if (version_compare(phpversion(), '7.0.0-dev', '<')) {
```

Comment:
my bad, for some reason I thought that we still have the function but calling it triggers a fatal. not sure why did I remember that, maybe because we kept the get_magic_quotes_runtime() but with hard-coded FALSE.